website/source/docs/guides/monitoring-telegraf.html.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: "docs"
+page_title: "Monitoring Consul with Telegraf"
+sidebar_current: "docs-guides-monitoring-telegraf"
+description: |-
+  Best practice approaches for monitoring a production Consul cluster with Telegraf.
+---
+
+# Monitoring Consul with Telegraf
+
+Consul has a number of metrics available through http endpoints. Measuring these metrics is useful for both health checking and debugging of cluster issues.
```

Comment:
Not just HTTP, though they are in the case of Prometheus or just API. In this case statsd is not retrieving from the HTTP endpoint. You might want to reword to:

> Consul makes available a range of metrics in various formats in order to measure the health and stability of a cluster, and diagnose or predict potential issues.

Reply:
Nice wording, fixed